### PR TITLE
Fix chain driver

### PR DESCRIPTION
**Pull request by pablomendezroyo**
## Context

The changes implemented to get dynamically the chain driver based on port and service name was breaking the old hardcoded way of defining the chain.

## Approach

Fix chain driver. There was referring to `.driver` instead of `.chain`. This should be compatible with both methods:
```
"chain" : "ethereum2-beacon-chain"
```
and 
```
  "chain": {
    "driver": "ethereum-beacon-chain",
    "serviceName": "beacon-chain",
    "portNumber": 3500
  }
```

## Test instructions

Install valid dappnode packages with both options in the manifest and maks sure that the chain is shown in the dashboard

### CODE REVIEW

File: packages/dappmanager/test/modules/compose/labelDb.test.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ describe("Parse and validate manifest labels to be used in the compose", () => {
       "dappnode.dnp.dependencies": '{"oneDependency": "dependency-name"}',
       "dappnode.dnp.avatar": "avatar-url",
       "dappnode.dnp.origin": "origin-url",
-      "dappnode.dnp.chain": '{"driverd": "ethereum2-beacon-chain-prysm"}',
+      "dappnode.dnp.chain": '{"driverd": "ethereudm2-beacon-chain-prysm"}',
```
Comment (dapplion) [inline packages/dappmanager/test/modules/compose/labelDb.test.ts:112]: Typo, must be `ethereum2` not `ethereudm2`

Comment (pablomendezroyo) [reply packages/dappmanager/test/modules/compose/labelDb.test.ts:112]: It is an intended typo to force another situation for the test

Comment (alexpeterson91) [reply packages/dappmanager/test/modules/compose/labelDb.test.ts:112]: ah ok, thats why i just did a PR to your branch and didnt merge after it passed tests.

Comment (dapplion) [reply packages/dappmanager/test/modules/compose/labelDb.test.ts:112]: Then please add a comment above this line

File: packages/dappmanager/test/modules/docker/parseContainerInfo.test.ts [hunk 2]
```diff
@@ -398,7 +398,7 @@ describe("modules / docker / parseContainerInfo", function () {
         dependencies: {},
         avatarUrl: "/ipfs/QmQnHxr4YAVdtqzHnsDYvmXizxptSYyaj3YwTjoiLshVwF",
         origin: "/ipfs/QmeBfnwgsNcEmbmxENBWtgkv5YZsAhiaDsoYd7nMTV1wKV",
-        chain: "ethereum",
+        chain: { driver: "ethereum" },
```
Comment (dapplion) [inline packages/dappmanager/test/modules/docker/parseContainerInfo.test.ts:401]: Why this change? `chain: "ethereum"` must be valid syntax

Comment (pablomendezroyo) [reply packages/dappmanager/test/modules/docker/parseContainerInfo.test.ts]: @dapplion fixed on the latest commit!

File: packages/dappmanager/src/modules/chains/getChainDriverName.ts [hunk 3]
```diff
@@ -17,6 +17,8 @@ const knownChains: { [dnpName: string]: ChainDriver } = {
   "kovan.dnp.dappnode.eth": "ethereum",
   "bitcoin.dnp.dappnode.eth": "bitcoin",
   "monero.dnp.dappnode.eth": "monero",
-  "prysm.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm"
+  "prysm.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "prysm-prater.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "lighthouse-prater.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "teku-prater.dnp.dappnode.eth": "ethereum-beacon-chain"
```
Comment (dapplion) [inline packages/dappmanager/src/modules/chains/getChainDriverName.ts:23]: Can you delete this file in another PR? I'm tired of everyone commiting to this file :rofl:

Comment (dapplion) [reply packages/dappmanager/src/modules/chains/getChainDriverName.ts:23]: Done in https://github.com/dappnode/DNP_DAPPMANAGER/pull/1021
